cEP-0018.md

```diff
@@ -22,6 +22,9 @@ they can write advanced linting bears. This will aim at supporting the more
 flexible visitor based method of AST traversal (as opposed to listener based
 mechanisms).
 
+Since ANTLR has poor support for retrieving original source code from the
```

Comment:
Can you give an example of where it is difficult to obtain the original source code?  You did investigations so incorporating an example should help readers understand why this is the case.